Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM python:3.9.16 AS py-deps
-COPY ./Pipfile ./Pipfile.lock ./
-RUN pip install -U pip setuptools pipenv && \
-    pipenv install --deploy --system --ignore-pipfile --clear
+
+RUN pip install -U pip setuptools wheel
+RUN pip install pdm
+
+
+WORKDIR /python
+COPY pyproject.toml pdm.lock README.md ./
```

Comment:
I dunno, PDM got mad at me when it wasn't there. I can test it again, I might have gotten my wires crossed when testing

Reply:
Yeah i dunno, `pdm sync` seems to fail when the README isn't present

<img width="1034" alt="Screenshot 2023-08-22 at 4 13 37 PM" src="https://github.com/Jellyfish-AI/jf_agent/assets/103140978/46e08243-efc6-4511-ba5e-8fbc05a554f4">

Reply:
weirdo, not a huge deal

Reply:
This is because we reference the README file in the project manifest ([here](https://github.com/Jellyfish-AI/jf_agent/pull/259/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R72)), and pdm requires it to be present-if-referenced ([discussion](https://github.com/pdm-project/pdm/issues/1990))